tests/vars.sample.yaml

```diff
@@ -1,5 +1,9 @@
 # Path to install_yamls repo
 install_yamls_path: ~/install_yamls #CUSTOMIZE_THIS
+
+# This flag signifies if TLS Everywhere is enabled on the source cloud
+enable_tlse: false
```

Comment:
you need to add it here: https://github.com/rdo-infra/rdo-jobs/blob/master/playbooks/data_plane_adoption/vars.yaml

Reply:
Yep, I already created a review https://review.rdoproject.org/r/c/rdo-jobs/+/52626